public/index.html

```diff
@@ -1267,6 +1267,37 @@ <h4 class="wide100p textAlignCenter">
                                             </div>
                                         </div>
                                     </div>
+                                    <div data-newbie-hidden data-tg-type="mancer, ooba, koboldcpp, aphrodite, tabby" name="dryBlock" class="wide100p">
+                                        <h4 class="wide100p textAlignCenter">
+                                            <label data-i18n="DRY Sampling">DRY Sampling</label>
```

Comment:
```suggestion
                                            <label data-i18n="DRY Repetition Penalty">DRY Repetition Penalty</label>
```

While DRY is of course technically a sampler, this makes it more clear what its purpose is.